File: datasets/IR-Plag/original-java-rest-kotlin/case-04/plagiarized/L5/05/pengulangan.kt
package IR

internal object inputoutput {
    fun hitHasil(m: Int) {
        for (a in m..10) {
            println(a.toString() + "\t\t" + a * 1.609)
            //ditambahin iteratornya
        }
    }

    @JvmStatic
    fun main(args: Array<String>) {
        val m = 1
        // print seusatu
        println("Miles\t\tKilometers")
        println("-------------------------------")
        // pake pengulangan while
        hitHasil(m)
    }
}

File: datasets/IR-Plag/convertai-original-java-rest-python/case-04/plagiarized/L4/05/pengulangan.py
import math

def hit_hasil(m):
    while m <= 10:
        print(f"{m}\t\t{m * 1.609}")
        m += 1

def main():
    m = 1
    print("Miles\t\tKilometers")
    print("-------------------------------")
    hit_hasil(m)

if __name__ == "__main__":
    main()

